regionmask/tests/test_unstructured.py

```diff
@@ -0,0 +1,21 @@
+import numpy as np
+import pytest
+import xarray as xr
+
+from .utils import dummy_region, expected_mask_2D
+
+
+@pytest.mark.parametrize("method", ["shapely", "pygeos"])
+def test_unstructured_dummy(method):
+    """Test for unstructured output."""
+    lat = [0.5, 0.5, 1.5, 1.5]
+    lon = [0.5, 1.5, 0.5, 1.5]
+    grid = xr.Dataset(coords={"lon": ("cells", lon), "lat": ("cells", lat)})
+
+    result = dummy_region.mask(grid, method=method, wrap_lon=False)
```

Comment:
`wrap_lon` must be set False

Reply:
mentioned also in `whats new`, or shall we try to set this automatically to `False` when `is_unstructured`?

Reply:
Good point, forgot about this one. I suggest we keep `wrap_lon` as is but skip this test when `is_unstructured`, i.e. update this line:

https://github.com/regionmask/regionmask/blob/b22c02c10b1ee2ea3ec29e426ad823360e30a39f/regionmask/core/utils.py#L103

to

```python
    if new_lon.ndim == 1 and not is_unstructured:
        ...
```

Reply:
```suggestion
    result = dummy_region.mask(grid, method=method)
```

Reply:
locally this failed

Reply:
```
>       result = dummy_region.mask(grid, method=method)

regionmask/tests/test_unstructured.py:17:
_ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _
regionmask/core/regions.py:289: in mask
    return _mask_2D(
regionmask/core/mask.py:214: in _mask_2D
    mask = _mask(
regionmask/core/mask.py:184: in _mask
    mask = _mask_edgepoints_shapely(mask, lon, lat, outlines, numbers)
_ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _

mask = array([ 0., nan,  1., nan]), lon = array([0.5, 1.5, 0.5, 1.5]), lat = array([0.5, 0.5, 1.5, 1.5])
polygons = [<shapely.geometry.polygon.Polygon object at 0x7f7f69abd0a0>, <shapely.geometry.polygon.Polygon object at 0x7f7f69abd5e0>, <shapely.geometry.polygon.Polygon object at 0x7f7f69abd430>]
numbers = [0, 1, 2], fill = nan

    def _mask_edgepoints_shapely(mask, lon, lat, polygons, numbers, fill=np.NaN):

        import shapely.vectorized as shp_vect

        # not sure if this is really necessary
        lon, lat, numbers = _parse_input(lon, lat, polygons, fill, numbers)

        LON, LAT, out, shape = _get_LON_LAT_out_shape(lon, lat, fill)

        mask = mask.flatten()
        mask_unassigned = np.isnan(mask)

        # find points at -180°E/0°E
        if lon.min() < 0:
            LON_180W_or_0E = np.isclose(LON, -180.0) & mask_unassigned
        else:
>           LON_180W_or_0E = np.isclose(LON, 0.0) & mask_unassigned
E           ValueError: operands could not be broadcast together with shapes (16,) (4,)